tools/depends/target/pythonmodule-pil/pillow-crosscompile.patch

```diff
@@ -3,7 +3,7 @@
 @@ -156,6 +156,16 @@
  
          _add_directory(include_dirs, "libImaging")
- 
+
```

Comment:
I didn't add these, these were already there when I patched the Makefile.pre.in again